crate/operator/bootstrap.py

```diff
@@ -332,6 +334,15 @@ async def bootstrap_cluster(
     # We first need to set the license, in case the CrateDB cluster
     # contains more nodes than available in the free license.
 
+    await send_operation_progress_notification(
+        namespace=namespace,
+        name=name,
+        message="Cluster creation started. Waiting for the node(s) to be created, "
+        "try to set license and create the necessary users.",
```

Comment:
Licenses are deprecated :) Maybe something like this?

```suggestion
        message="Cluster creation started. Waiting for the node(s) to be created "
        "and creating other required resources.",
```